### DISCUSSION TITLE
Add "Edit this page on GitHub" to documentation

### DISCUSSION THREAD

**Discussion by samuliasmala** [chakra-ui/chakra-ui-docs#680] [Ideas]
I propose adding "Edit this page on GitHub" link to each documentation page in the same style Next.js documentation have (please check e.g. the bottom of the [Next.JS router doc page](https://nextjs.org/docs/api-reference/next/router#typescript)). Should be quite straightforward to add, but what do you think? Would PR be welcome?

**Comment by nikolovlazar** [comment]
Of course @samuliasmala! If you're up for it, you can implement this and open a PR. Thank you!

**Reply by samuliasmala** [reply]
Great! I just [created a PR](https://github.com/chakra-ui/chakra-ui-docs/pull/708).
